engine/src/main/java/org/camunda/bpm/engine/impl/cfg/ProcessEngineConfigurationImpl.java

```diff
@@ -4961,7 +4961,7 @@ public void setHistoryCleanupBatchThreshold(int historyCleanupBatchThreshold) {
   }
 
   public boolean isHistoryCleanupMetricsEnabled() {
```

Comment:
🔧 Maybe we could add a little javadoc to explain that it's not just a simple getter anymore.

Reply:
Added javadoc